### ISSUE TITLE
3주차 이미지 첨부용 이슈

### ISSUE THREAD

**Issue by sugarghost** [sugarghost/hanghae99_AI_PLUS_2#1]
![Untrained Model F1-Score](https://github.com/user-attachments/assets/744aad34-56e1-40c2-875e-3af684f7dd7f)
![Pre-trained Model F1-Score](https://github.com/user-attachments/assets/4c13c2c7-533f-4c97-bab8-00dd12428441)
![Training Loss per Epoch](https://github.com/user-attachments/assets/88b6ae57-ec11-4637-be08-8764ea97d4cd)
